### DISCUSSION TITLE
Patching patches inside textboxes

### DISCUSSION THREAD

**Discussion by samvanemelen** [dolanmiu/docx#2279] [Q&A] [answered]
Hi All! I am absolutely loving the versatility of this library, but stumbled upon a challenge recently. I have a word template, where all text is in dedicated textboxes rather than inline in the document. Is the patcher capable of replacing patches inside textboxes? By my initial tests it doesn't seem that way. Sadly I cannot remake the template so I would love if I could find a way to get this to work.

**Comment by dolanmiu** [comment, marked as answer]
At the moment, they're not supported nope! Will be prioritised
